code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -480,27 +515,30 @@
 	category = "Defensive"
 	cost = 1
 
+//THESE ARE NOT PURCHASABLE SPELLS! They're references to old spells that got removed + shit that sounds stupid but fun so we can painfully lock behind a dimmer component
+
+/datum/spellbook_entry/challenge
+	name = "Take the Challenge"
+	refundable = FALSE
+	category = "Challenges"
+	buy_word = "Accept"
+
+/datum/spellbook_entry/challenge/multiverse
+	name = "Multiverse Sword"
+	desc = "The Station gets a multiverse sword to stop you. Can you withstand the hordes of multiverse realities?"
+
+/datum/spellbook_entry/challenge/antiwizard
+	name = "Friendly Wizard Scum"
+	desc = "A \"Friendly\" Wizard will protect the station, and try to kill you. They get a spellbook much like you, but will use it for \"GOOD\"."
```

Comment:
BASED

Reply:
wait, uh

won't the good witch spawn on the wizard ship with you and promptly be ganked?

Reply:
this should also probably mention "inviting" a friendly wizard instead of just saying that one will be there. I like the idea of the friendly wizard not actually giving a shit about the station, but instead just taking on the challenge of defending it from you as a form of sport.

Reply:
These are pure flavor, they don't do anything on their own I don't think

Reply:
You know for all that reviewing you do ATHATH you sure don't know how to read the comment something like 8 lines up that says "THESE ARE FLAVOR"